seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -2419,6 +2427,10 @@
         "",
         "// Only change code below this line",
         "",
+        "",
+        "// Only change code above this line",
+        "",
+        "timesFive(5);",
```

Comment:
I think we might be talking a little bit at cross purposes. You are absolutely right that until the camper defines timesFive, it'll throw an error. the way I was looking at it was the function call to help with testing and debugging that is built into most of the later challenges and thus would not need to be deleted at all. That being said, I can see how it might be confusing to someone inexperienced with functions. I'll go ahead and remove them.